Tasks/PipAuthenticateV1/utilities.ts

```diff
@@ -0,0 +1,14 @@
+import * as url from 'url';
+import * as tl from 'azure-pipelines-task-lib/task';
+
+export function formPipCompatibleUri(userName: string, password: string, uri: string): string{
```

Comment:
Is there anything special pip requires? Doesn't look like it, possibly rename to "addCredentialsToUri" or similar?

Reply:
cool, will do.